.github/workflows/create-buildpack-release.yaml

```diff
@@ -53,38 +53,51 @@ jobs:
           path: ./buildpacks/${{ github.event.inputs.buildpack }}/VERSION
           release-type: ${{ github.event.inputs.release-type }}
           pre-release: ${{ github.event.inputs.pre-release-label }}
+          version: ${{ steps.version.outputs.version }}
       - name: Bump Version
         run: |
-          printf "${{ steps.version.outputs.version }}" > ./buildpacks/${{ github.event.inputs.buildpack }}/VERSION
+            printf  $version > $path
       - name: Run Tests
+        with:
+          target: buildpacks.${{ github.event.inputs.buildpack }}.tests
         run: |
-          make buildpacks.${{ github.event.inputs.buildpack }}.tests
+          make $target
       - name: Publish Container
+        with:
+          url: ${{ env.DEPENDENCY_BASE_URL }}
+          registry: ${{ env.REGISTRY }}/${{ env.CONTAINER_PATH }} buildpacks.${{ github.event.inputs.buildpack }}.images.publish
         run: |
-          make base_url=${{ env.DEPENDENCY_BASE_URL }} registry.location=other REGISTRY=${{ env.REGISTRY }}/${{ env.CONTAINER_PATH }} buildpacks.${{ github.event.inputs.buildpack }}.images.publish
+          make base_url=$url registry.location=other REGISTRY=$registry
       - name: Prepare Commit
+        with:
+          target: buildpacks.${{ github.event.inputs.buildpack }}.commit.prep
         run: |
-          make buildpacks.${{ github.event.inputs.buildpack }}.commit.prep
+          make $target
       - name: Commit
+        with:
+          version: ./buildpacks/${{ github.event.inputs.buildpack }}/VERSION
+          buildpack: ./buildpacks/${{ github.event.inputs.buildpack }}/buildpack.toml
+          message: Update ${{ github.event.inputs.buildpack }} buildpack version to ${{ steps.version.outputs.version }}
         run: |
-          git add ./buildpacks/${{ github.event.inputs.buildpack }}/VERSION
-          git add ./buildpacks/${{ github.event.inputs.buildpack }}/buildpack.toml
+          git add $version
+          git add $buildpack
           git config user.name github-actions
           git config user.email github-actions@github.com
-          git commit -m 'Update ${{ github.event.inputs.buildpack }} buildpack version to ${{ steps.version.outputs.version }}'
+          git commit -m '$message'
           git push
       - name: Create Tag
         id: tag
+        with:
+          tag: ${{ github.event.inputs.buildpack }}-buildpack/v${{ steps.version.outputs.version }}
         run: |
-          tag=${{ github.event.inputs.buildpack }}-buildpack/v${{ steps.version.outputs.version }}
           git tag $tag
           git push --tags
           echo "::set-output name=tag::$tag"
       - id: artifacts
+        with:
```

Comment:
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable suggests that this should use `env:`, not `with:`